### DISCUSSION TITLE
Filters Don't Work Until Refresh of Page

### DISCUSSION THREAD

**Discussion by dennisguilbault** [code4recovery/12-step-meeting-list#590] [Q&A] [answered]
I've been having a problem for some time now that when I change a filter it won't show the new results until I refresh the page with F5.

**Comment by tech2serve** [comment, marked as answer]
Please [read this](https://github.com/code4recovery/12-step-meeting-list/discussions/412#discussion-3447102) and edit your question in order for us to help you better.

I will say the issue you’ve described occurs most often for users when there is a conflict being caused by something else on your site. If you search the discussions (or issues) here, you’ll see a number of other similar topics (i.e.,, unexpected filtering behaviors) and by looking in the dev/debugging console of your web browser, or disabling other plugins, you’ll likely isolate the cause and be able to fix things.

If you’re still stuck, please edit your question and we’ll take a closer look.

Yis, Tim

**Reply by dennisguilbault** [reply]
I deleted the TablePress plug-in and my filtering is working now.
Thanks!

**Comment by dennisguilbault** [comment]
Well, that same behavior is back again.  I really don't know what I'd be looking at in "debug" mode.  I got a new laptop.  Maybe some sort of browser setting for Chrome???

**Reply by joshreisner** [reply]
hi @dennisguilbault what’s your website url?

**Comment by dennisguilbault** [comment]
aatopeka.org

On Sat, Nov 12, 2022 at 10:19 AM Josh Reisner ***@***.***>
wrote:

> hi @dennisguilbault <https://github.com/dennisguilbault> what’s your
> website url?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/590#discussioncomment-4124176>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWRSKR7OTGXMAY7HCWBRUYLWH67R7ANCNFSM5KDQGLKA>
> .
> You are receiving this because you were mentioned.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/590/comments/4124176@
> github.com>
>


-- 
Dennis Guilbault
(785) 806-2255

*"You Can't Always Get What You Want, But If You Try Sometimes, You Get
What You Need"*
*-Rolling Stones*

**Reply by joshreisner** [reply]
thanks @dennisguilbault - i see the problem on the `https` version but not the `http` version. the console log says `Blocked loading mixed active content “[http](http://www.aatopeka.org/wp-admin/admin-ajax.php?action=tsml_groups)"`

i think the issue is the plugin thinks the website address should be `http` when it is not. go to wordpress settings and see if the Site Address and WordPress Address have the https version in there. if not, try adding the `s` there?

**Comment by dennisguilbault** [comment]
Ok, I'll check it out.  Thanks

On Sat, Nov 12, 2022 at 10:43 AM Josh Reisner ***@***.***>
wrote:

> thanks @dennisguilbault <https://github.com/dennisguilbault> - i see the
> problem on the https version but not the http version. the console log
> says Blocked loading mixed active content “[http](
> http://www.aatopeka.org/wp-admin/admin-ajax.php?action=tsml_groups)"
>
> i think the issue is the plugin thinks the website address should be http
> when it is not. go to wordpress settings and see if the Site Address and
> WordPress Address have the https version in there. if not, try adding the
> s there?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/590#discussioncomment-4124278>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWRSKR5RHAZ4WGJHPM2UPELWH7CJXANCNFSM5KDQGLKA>
> .
> You are receiving this because you were mentioned.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/590/comments/4124278@
> github.com>
>
-- 
Dennis Guilbault
(785) 806-2255

*"You Can't Always Get What You Want, But If You Try Sometimes, You Get
What You Need"*
*-Rolling Stones*

**Comment by dennisguilbault** [comment]
We’ll that didn’t work and now I can’t get to my admin site

On Sat, Nov 12, 2022 at 10:43 AM Josh Reisner ***@***.***>
wrote:

> thanks @dennisguilbault <https://github.com/dennisguilbault> - i see the
> problem on the https version but not the http version. the console log
> says Blocked loading mixed active content “[http](
> http://www.aatopeka.org/wp-admin/admin-ajax.php?action=tsml_groups)"
>
> i think the issue is the plugin thinks the website address should be http
> when it is not. go to wordpress settings and see if the Site Address and
> WordPress Address have the https version in there. if not, try adding the
> s there?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/590#discussioncomment-4124278>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWRSKR5RHAZ4WGJHPM2UPELWH7CJXANCNFSM5KDQGLKA>
> .
> You are receiving this because you were mentioned.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/590/comments/4124278@
> github.com>
>
-- 
Dennis Guilbault
(785) 806-2255

*"You Can't Always Get What You Want, But If You Try Sometimes, You Get
What You Need"*
*-Rolling Stones*

**Reply by joshreisner** [reply]
@dennisguilbault yes, it makes sense you would need to log in again. try logging in the second time at https://www.aatopeka.org/wp-login.php

i do see the error still on https. do you have a caching plugin installed?

**Reply by joshreisner** [reply]
@dennisguilbault if you feel like you need to undo that change and you cant gain access back to the admin panel, you can also override that setting by adding this to your wp-config.php:

```
define( 'WP_HOME', 'http://www.aatopeka.org' );
define( 'WP_SITEURL', 'http://www.aatopeka.org );
```

but i would only use that as a last resort. as i mentioned in my last message, it makes sense you would need to log in again. try logging in the second time at https://www.aatopeka.org/wp-login.php

i do see the error still on https. do you have a caching plugin installed? perhaps refreshing the cache would help.

at any rate, the issue is the website is not sure what its proper url is supposed to be.

another thing to perhaps try, would be to use a plugin like "Force SSL" to make sure users always get the https version. it's possible this will also make the site aware of its proper URL: https://wordpress.org/plugins/wp-force-ssl/
